packages/core/src/__tests__/ceramic-api.test.ts

```diff
@@ -1,9 +1,9 @@
 import { jest } from '@jest/globals'
 import tmp from 'tmp-promise'
-import type { Ceramic } from '../ceramic.js'
+import { Ceramic } from '../ceramic.js'
```

Comment:
Please, get the imports in a previous shape if you do not use the changes in the actual code below.

Reply:
This is an unused import.